test/common/network/connection_impl_test.cc

```diff
@@ -2048,6 +2051,68 @@ TEST_P(ConnectionImplTest, FlushWriteAndDelayCloseTest) {
   server_connection_->close(ConnectionCloseType::FlushWriteAndDelay);
   dispatcher_->run(Event::Dispatcher::RunType::Block);
 }
+
+// Test that remote early close is not managed by the filter manager.
+TEST_P(ConnectionImplTest, FlushWriteAndDelayCloseWithoutFilterManager) {
+  TestScopedRuntime scoped_runtime;
+  scoped_runtime.mergeValues(
+      {{"envoy.reloadable_features.connection_close_through_filter_manager", "true"}});
+  setUpBasicConnection();
+  connect();
+
```

Comment:
Hmmm, the test name is a little weird for me. The name and comment tell me the case is used to test the case where the closing won't be taken over. But the actually logic is reversed?

Reply:
changed the name and remote early close is triggered during the delay closing after flush, in normal case, there will also be data loss if we are delaying close.